packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -242,6 +245,19 @@ export class RemoteChannelContext implements IChannelContext {
         return this.summarizerNode.getGCData();
     }
 
+    public updateUsedRoutes(usedRoutes: string[]) {
+        /**
+         * Currently, DDSs are always considered referenced and are not garbage collected. Update the summarizer node's
+         * used routes to contain a route to this channel context.
+         * Once we have GC at DDS level, this will be updated to use the passed usedRoutes. See -
+         * https://github.com/microsoft/FluidFramework/issues/4611
+         */
+        // back-compat: 0.33 - updateUsedRoutes is added in 0.33. Remove the check here when N >= 0.36.
+        if (this.summarizerNode.updateUsedRoutes !== undefined) {
+            this.summarizerNode.updateUsedRoutes([""]);
+        }
```

Comment:
Just an FYI, this can be done in a single line using `?.`:
```suggestion
        this.summarizerNode.updateUsedRoutes?.([""]);
```

But totally understand if you want to keep it separate for clarity around back-compat.

Reply:
I did this and the compiler was complaining about unbound methods!